packages/shell-api/src/database.ts

```diff
@@ -1429,6 +1437,7 @@ export default class Database extends ShellApiWithMongoClass {
         CommonErrors.CommandFailed
       );
     }
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
autoformat kept removing the `as Document[]` for me so I had to add this comment